src/main/java/org/broadinstitute/hellbender/tools/exome/FilterTargets.java

```diff
@@ -317,9 +330,159 @@ public String reasonToFilter(final Target target) {
     }
 
     /**
+     * Class that represent a interval of targets to exclude.
+     *
+     * It contains a copy of the interval specification (e.g. "chr1", "chr1:1+", "chr2:100,000,000-200,000,000").
+     *
+     * And the location where it was specified so that we can provide more informative error and target rejection messages.
+     */
+    private final class ExcludeInterval implements Locatable {
```

Comment:
I don't insist.  My goal in code review is to point out other possibilities (even pure opinion) for you to choose from *if you want* i.e. to be an extra set of eyes but not in the sense of "Big Brother is watching."